modules/extensions/partials/ref_the-woopra-telemetry-plugin.adoc

```diff
@@ -0,0 +1,21 @@
+[id="the-woopra-telemetry-plugin"]
+
+= The Woopra Telemetry Plugin Yo
+
+The link:https://github.com/che-incubator/che-workspace-telemetry-woopra-plugin[Woopra Telemetry Plugin] is a plugin built to send telemetry from link:https://che.openshift.io[Hosted Che] to Segment and Woopra. The plugin has no dependencies other than a valid Woopra domain and Segment Write key. The link:https://raw.githubusercontent.com/che-incubator/che-workspace-telemetry-woopra-plugin/master/meta.yaml [plugin's meta.yaml] file has 5 environment variables that can be passed to the plugin:
+
+- `CHE_API` - This is set to the value of `$(CHE_API_INTERNAL)` and should not change.
```

Comment:
I still wondering if we really need this env var ? probably we should create an issue to remove it ?

Reply:
I agree.  The plugin uses `${CHE_API}` to get the Workspace from the API, but in a workspace it should have access to `${CHE_API_INTERNAL}`